configuration/raspi-config.md

```diff
@@ -79,7 +79,7 @@ From here you can change what happens when your Pi boots. Use this option to cha
 <a name="localisation-options"></a>
 ### Localisation Options
 
-The localisation submenu gives you these options to choose from: keyboard layout, time zone, locale, and WiFi country code. All options on these menus default to British or GB until you change them.
+The localisation submenu gives you these options to choose from: keyboard layout, time zone, locale, and wireless LAN country code. All options on these menus default to British or GB until you change them.
```

Comment:
Wireless country code is now unset at install time - doesn't default to GB anymore.

Reply:
(raspi-config UI says 'timezone' instead of 'time zone'. Lexico.com/en doesn't have a entry for timezone with no space).